deploy/scripts/src/mobile/test.ts

```diff
@@ -19,9 +20,14 @@ import {
   checkWorkingCameraRedirect,
   getBiometricToken,
   postFinishBiometricToken,
-  getSessionId,
-  setSessionCookie,
-  doAuthorizeRequest
+  checkIdCheckAppRedirect,
```

Comment:
these check functions imply they are validation steps. From my understanding, they:

1) Do a network request
2) Perform validation

Is there a way to rename this so it's a bit clearer what it is doing?

Reply:
Have updated the naming convention of these to `post{endpointName}AndValidateRedirect`